### PR TITLE
Use python3 print function in jupyter notebook

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

### CODE REVIEW

**Comment by jantonguirao** [comment]
Build #607372

File: docs/examples/extend/create_a_custom_operator.ipynb [hunk 1]
```diff
@@ -186,6 +186,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from __future__ import print_function\n",
```
Comment (JanuszL) [inline docs/examples/extend/create_a_custom_operator.ipynb]: I don't know if we need that one. I think Jupyter deals with that without that import, doesn't it?

Comment (Kh4L) [reply docs/examples/extend/create_a_custom_operator.ipynb]: They seem to be using it in their own doc, maybe it's safe to have it anyway https://jupyter-notebook.readthedocs.io/en/stable/examples/Notebook/Running%20Code.html#sys.stdout-and-sys.stderr

Comment (jantonguirao) [reply docs/examples/extend/create_a_custom_operator.ipynb]: removed

**Pull request merged**